src/Core/Utils/StackTrace.cpp

```diff
@@ -1,29 +1,12 @@
 #include <Core/Utils/StackTrace.hpp>
-
-#ifndef OS_WINDOWS
-
-// A C++ function that will produce a stack trace with demangled function and method names.
-#    include <cxxabi.h>   // for __cxa_demangle
-#    include <dlfcn.h>    // for dladdr
-#    include <execinfo.h> // for backtrace
-#    include <sstream>
-
-#    if defined( COMPILER_CLANG )
-#        pragma clang diagnostic ignored "-Wformat"
-#        pragma clang diagnostic ignored "-Wformat-extra-args"
-#    elif defined( COMPILER_GCC )
-#        pragma GCC diagnostic ignored "-Wformat"
-#        pragma GCC diagnostic ignored "-Wformat-extra-args"
-#    endif
-
-#endif
+#include "radium_backtrace.h" // for backtrace, generated by cmake
```

Comment:
Is this really portable ? Is the path correct in the install tree ?

Reply:
Yes, this is correct. This is a private include used only in a .cpp source file. So this header is not installed because not exposed to the public interface of Radium.

Reply:
RIght, my bad.
The CI fails however.

Reply:
I saw ... I'll fix it